gb.yaml

```diff
@@ -47,6 +47,19 @@ months:
     regions: [gb]
     week: 1
     wday: 1
+    year_ranges:
+      until: 2019
+  - name: May Day 
```

Comment:
Perfect.:)
year_ranges feature suit fot this use-case.

Reply:
Thanks for reviewing :)